### DISCUSSION TITLE
有表单联动的情况下，如何获取到当前显示的表单结构

### DISCUSSION THREAD

**Discussion by clarastar** [alibaba/formily#2533] [Q&A 1.x] [answered]
https://formilyjs.org/#/bdCRC5/dzUZU8il

表单有很多字段，除了用 getFieldState('*').visible 这个方法，循环每个字段来获取到当前显示的字段外，是否还有别的方法呢？

**Comment by janryWang** [comment, marked as answer]
getFormGraph或者form.fields都行
